docs/cookbook/fml/growable-collections.mdx

```diff
@@ -0,0 +1,247 @@
+---
+id: growable-collections
+title: "Managing collections not known before release"
+slug: /cookbook/fml/growable-collections
+---
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Problem
+
+You want to configure a collection of things of the same type, but the configuration isn't known at build time.
+
+For example,
+
+ * a collection of messages, or
+ * a collection of wallpapers.
+
+## Solution
+
+Use a type of `Map<String, T>`.
+
+```yaml
+---
+channels:
+  - debug
+  - release
+features:
+  theming-feature:
+    description: Configuration for theming feature.
+    variables:
+      asset-urls:
+        description: A collection of downloadable assets
+        type: Map<String, String>
+        default: {}
+  defaults:
+    channel: debug
+    value:
+      assets-urls:
+        kittens: https://placekitten.com/600/900
+        bill-murray: https://www.fillmurray.com/600/900
+        flickr: https://loremflickr.com/600/900
+    channel: release
+    value:
```

Comment:
Is the indentation correct here? Should `value` be indented on more level?

Reply:
You're right!